### DISCUSSION TITLE
2006/08/12/%E7%94%A8-labview-%E7%BC%96%E5%86%99-wizard-%E7%B1%BB%E5%9E%8B%E7%9A%84%E5%BA%94%E7%94%A8%E7%A8%8B%E5%BA%8F-2-labview-6-1-7-1

### DISCUSSION THREAD

**Discussion by giscus** [ruanqizhen/blog#106] [Announcements]
# 2006/08/12/%E7%94%A8-labview-%E7%BC%96%E5%86%99-wizard-%E7%B1%BB%E5%9E%8B%E7%9A%84%E5%BA%94%E7%94%A8%E7%A8%8B%E5%BA%8F-2-labview-6-1-7-1

四、Tab 控件+事件处理结构

https://blog.qizhen.xyz/2006/08/12/%E7%94%A8-labview-%E7%BC%96%E5%86%99-wizard-%E7%B1%BB%E5%9E%8B%E7%9A%84%E5%BA%94%E7%94%A8%E7%A8%8B%E5%BA%8F-2-labview-6-1-7-1

**Comment by ruanqizhen** [comment]
之前博客下的留言：

colin
2006年08月14日 22:20 编辑
您是高手啊~~~~崇拜！！！！

回复
Qizhen
2006年08月14日 18:33 编辑
我去看了一下，这个 blog 确实做得满漂亮的。不过它的作者不是会更清楚它是如何做出来的吗:)
我的想法是，作这样一个 blog 首先由有素材，要有哪些漂亮的图片。
你现在把它上面的图片选中，Ctrl+C，再到自己的帖子上 Ctrl+V。也可以做出和它差不多的页面了 :D

回复
binglan
2006年08月12日 23:15 编辑
先给你一个BLOG，很漂亮，人也很漂亮啊！嘿嘿！我不知道在哪给你留言。就随便找了个地方。我想问问你他们怎么弄得。为什么问你呢，缘分！http://loveeven0401.spaces.live.com/

回复
